ethers-core/src/types/trace/mod.rs

```diff
@@ -11,7 +11,7 @@ pub use filter::*;
 mod geth;
 pub use geth::*;
 
-#[derive(Debug, Clone, Deserialize, Serialize)]
+#[derive(Debug, PartialEq, Clone, Deserialize, Serialize)]
```

Comment:
clippy will suggest +Eq
```suggestion
#[derive(Debug, PartialEq, Eq, Clone, Deserialize, Serialize)]
```